app/controllers/samples_controller.rb

```diff
@@ -779,6 +782,18 @@ def report_v2
     render json: PipelineReportService.call(pipeline_run.id, background_id)
   end
 
+  def amr
```

Comment:
Worth adding an rspec test for this?

Reply:
Will not add at this moment.
Did not change the code that generates it, and should be eventually reviewer.